pkg/types/baremetal/validation/platform.go

```diff
@@ -124,6 +124,16 @@ func validateOSImageURI(uri string) error {
 	return nil
 }
 
+func validateNTPServers(p *baremetal.Platform, fldPath *field.Path) (allErrs field.ErrorList) {
+	for _, ip := range p.AdditionalNTPServers {
+		if err := validate.IP(ip); err != nil {
```

Comment:
The ntp server could be either IP or server name, does this restrict the NTP server to an IP address?

Reply:
Everything else in those configs is IP addresses because it's configuring the things that DNS points to.
Users will absolutely want to be able to specify their NTP server by name.

Reply:
removed validation as ip